cfgov/unprocessed/css/enhancements/layout.less

```diff
@@ -311,6 +311,81 @@
 }
 
 /* topdoc
+  name: Adds block background that bleeds off page 
+  family: cf-layout
+  notes:
+    - ".block__bg-extend-right requires that overflow-x be set to 
+       hidden on a parent element."
+  tags:
+    - cf-layout
+*/
+
+.block {
```

Comment:
I can move them to a separate PR, but wouldn't it make sense to have context and a use case for the addition? The styles are also a fairly significant part of the search bar